api/src/App/Routes.php

```diff
@@ -93,6 +93,38 @@ function () use ($app, $authMiddleware) {
     }
 )->add(new App\Middleware\CiabMiddleware($app))->add($authMiddleware);
 
+$app->group(
+    '/payment',
+    function () use ($app, $authMiddleware) {
+        $app->add(function ($request, $response, $next) {
+            \Stripe\Stripe::setApiKey($_ENV['STRIPE_PRIVATE_KEY']);
+            return $next($request, $response);
+        });
+        $app->post('/checkout', 'App\Controller\Payment\PostCheckout');
+    }
+)->add(new App\Middleware\CiabMiddleware($app))->add($authMiddleware);
+
```

Comment:
We talked about moving the payment into a vendor driven sort of framework.

Is that a version next?

Reply:
Yeah, the rest of the payment code isn't even in here, honestly -- I held it out. So this section can just get cut for now.

Reply:
Sounds good. Lets cut this code and put it in the next chunk.